### DISCUSSION TITLE
cell.getParent() method return null of mindmap

### DISCUSSION THREAD

**Discussion by yogalin1993** [antvis/X6#2630] [Q&A]
https://x6.antv.vision/zh/examples/showcase/practices#mindmap
In the demo above, I got null when  I called cell.getParent() method of 'node:click' event, even if the node I clicked has parent on the truth. The code snippet:
```js
this.graph.on('node:click', ({ e, x, y, node, cell, view }) => {
  const parent = cell.getChildren()
  console.log('parent', parent) // null
})
```

**Comment by xiazhaohui** [comment]
i have same question
